eagleproject/core/views.py

```diff
@@ -335,3 +335,23 @@ def staking_stats(request):
             "lockupSum": float(total_staked) if total_staked else 0,
         }
         return JsonResponse(data)
+
+
+def staking_stats_by_duration(request):
+    with connection.cursor() as cursor:
+        query = """
+        select duration as "Duration", sum(amount) as "Total Staked" from core_ognstaked 
+        where 
+            is_staked = true 
+            and rate > 0 
+            and block_time + interval '1 day' * duration > now()
+        group by duration;
+        """
```

Comment:
This codebase supports both PostgreSQL and SQLite.  This query won't work for SQLite due to the date math.

Reply:
Just curious, aren't we using PostgreSQL on the deployed instance? Because I remember adding a similar query on Metabase for a different dashboard, it works. Seems like this new endpoint by @ZaHuPro will be used only for the website, so I don't think SQLite support is needed for this query (unless I'm missing something).

Reply:
If there are similar queries that use PostgreSQL-only features then that's a bug.  I didn't notice any using PG's intervals in my quick review.

This isn't worth arguing about.  I went ahead and took care of it myself.

@ZaHuPro make sure my changes still fit your needs and we can get this merged and deployed today.

Reply:
> I didn't notice any using PG's intervals in my quick review.

It's probably because it's on metabase and didn't need a review. 😅 
I will update them though

Reply:
Works fine